prov/shm/src/smr.h

```diff
@@ -206,6 +211,13 @@ struct smr_ep {
 #define smr_ep_rx_flags(smr_ep) ((smr_ep)->util_ep.rx_op_flags)
 #define smr_ep_tx_flags(smr_ep) ((smr_ep)->util_ep.tx_op_flags)
 
+static inline int mmap_shm_name(char *shm_name, const char *ep_name,
```

Comment:
Rename to smr_mmap_name for prefix consistency

Reply:
Done.